doc/source/examining/loading_data.rst

```diff
@@ -1788,6 +1788,12 @@ Generic AMR Data
 See :ref:`loading-numpy-array` and
 :func:`~yt.frontends.stream.data_structures.load_amr_grids` for more detail.
 
+.. note::
+
+   It is now possible to load data using only *callable* functions, rather
```

Comment:
"callable function" is a pleonasm. Maybe just say "function" here (even if "callable" by itself is the more technically correct term, it's also not as clear to most users)

Reply:
uh, just in case anybody besides me needs more information:

> ple·o·nasm
> /ˈplēəˌnazəm/
> [Learn to pronounce](https://www.google.com/search?sxsrf=ALiCzsZNFQ8TA2ZODgoV5SFrg0muHxaIrw:1659623425663&q=how+to+pronounce+pleonasm&stick=H4sIAAAAAAAAAOMIfcRoxS3w8sc9YSnDSWtOXmPU5uINKMrPK81LzkwsyczPExLhYglJLcoV4pHi4uIoyEnNz0sszrViUWJKzeNZxCqZkV-uUJKvUADUkw_UlKoAUwIA94fwUlsAAAA&pron_lang=en&pron_country=us&sa=X&ved=2ahUKEwjDj_2Fs635AhW8IDQIHWvjBKgQ3eEDegQIBRAK)
> noun
> the use of more words than are necessary to convey meaning (e.g. see with one's eyes ), either as a fault of style or for emphasis.

Reply:
oops, sorry, again a case of words-that-are-much-more-common-in-french 🙈